### PR TITLE
docs/usage-with-nuxt

### PR DESCRIPTION
**Pull request by jakubSzuminski**
Updated the `README.md` file with a short setup note when using Nuxt.js.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,13 +64,24 @@ Then, you've to register it in your Vue instance configuration, namely in `compo
...
 *NOTE:*
 *If you would like to use Highcharts-Vue wrapper by attaching it using `<script>` tag in your `<head>` section of HTML document, of course it is possible and you should use one of .js files from `dist` of this package directory. After that, the `HighchartsVue` object should be available from `window` scope. Here is the example with this way of implementation: [JSFiddle example](https://jsfiddle.net/BlackLabel/ukc2kqnb/)*
 
+### Using with NuxtJS
+Nuxt.js executes the code twice - first time on the server side and then on the client side. During the first run (on the server) the Highcharts object is not initialized as there is no `window` available. Thus, you should always check if Highcharts has already been initialized:
+```JavaScript
+import Highcharts from 'highcharts';
+import HighchartsExporting from 'highcharts/modules/exporting';
+
+if (typeof Highcharts === 'object') {
```
Comment (sjoshid) [inline README.md:80]: In Nuxt, if you want `window` to be available, then one of the client side only hooks (eg.,`onMounted()`) can be used. 

If you do, this condition is not required.

Comment (jakubSzuminski) [reply README.md:80]: Absolutely, the `onMounted` hook is only triggered on the client side when the `window` client is available. However, it's common to initialize the Highcharts modules (like the `exporting` module) when initializing the Vue application and installing the `highcharts-vue` plugin.

Comment (sjoshid) [reply README.md:80]: Yes but, we are talking about Nuxt section. Not Vue. 

In a normal Vue app without Nuxt, this condition is right. I'm not saying this is wrong in Nuxt, it's just that I haven't seen anyone initialize like that.

**Comment by sjoshid** [comment]
I added some comments as you already know.

My two cents: There is Nuxt 2 which is old and not favored anymore. Then there is Nuxt 3 which is greater and better. 

The section you added doesn't say which Nuxt version it applies to. This might add more confusion. 

If you think what you added applies to Nuxt 2, I'd recommend it be removed. As I said, Nuxt 2 is old. 

As for Nuxt 3, I can definitely say this lib needs more work.

**Pull request merged**